src/main/java/com/eprosima/fastdds/fastddsgen.java

```diff
@@ -541,43 +551,81 @@ private void showVersion()
         System.out.println(m_appName + " version " + version);
     }
 
+
+
+    /*
+     * ----------------------------------------------------------------------------------------
+     *
+     * Arguments
+     */
+    private static final String case_sensitive_arg = "-cs";
+    private static final String output_path_arg = "-d";
     private static final String default_container_prealloc_size = "-default-container-prealloc-size";
+    private static final String default_extensibility_arg = "-default_extensibility";
+    private static final String default_extensibility_short_arg = "-de";
+    private static final String specific_platform_arg = "-example";    
+    private static final String extra_template_arg = "-extrastg";
+    private static final String help_arg = "-help";
+    private static final String include_path_arg = "-I";
+    private static final String language_arg = "-language";
+    private static final String no_typesupport_arg = "-no-typesupport";
+    private static final String no_dependencies_arg = "-no-dependencies";
+    private static final String package_arg = "-package";
+    private static final String disable_preprocessor_arg = "-ppDisable";
+    private static final String preprocessor_path_arg = "-ppPath";
+    private static final String python_bindings_arg = "-python";
+    private static final String replace_arg = "-replace";
+    private static final String temp_dir_arg = "-t";
+    private static final String typeobject_arg = "-typeobject";
+    private static final String cnames_arg = "-typesc";
+    private static final String ros2_names_arg = "-typeros2";
+    private static final String fusion_arg = "-fusion";
+    private static final String version_arg = "-version";
+
+    private static final String generate_api_arg = "-genapi";
```

Comment:
Why these two lines are separated?

Reply:
If there is because they are develop argument, please add a comment before the first one.

Reply:
Fixed in [b895dac](https://github.com/eProsima/Fast-DDS-Gen/pull/275/commits/b895dac6bfd265916d48306d80f210087e16a199)